algorithms/kernel/objective_function/logistic_loss/logistic_loss_dense_default_batch_impl.i

```diff
@@ -499,7 +499,10 @@ services::Status LogLossKernel<algorithmFPType, method, cpu>::compute(NumericTab
 
             TArrayScalable<algorithmFPType, cpu> aX(n * p);
             TArrayScalable<algorithmFPType, cpu> aY(n);
-            s |= objective_function::internal::getXY<algorithmFPType, cpu>(dataNT, dependentVariablesNT, ntInd, aX.get(), aY.get(), nRows, n, p);
+            {
+                DAAL_ITTNOTIFY_SCOPED_TASK(getXY);
+                s |= objective_function::internal::getXY<algorithmFPType, cpu>(dataNT, dependentVariablesNT, ntInd, aX.get(), aY.get(), nRows, n, p);
```

Comment:
It is safer to call the function inside `DAAL_CHECK_STATUS_VAR()` instead of accumulating status in `s` variable without checking it